src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicWinForms.cs

```diff
@@ -96,18 +96,18 @@ public void Change_Control_Property_In_Code()
         [Fact, Trait(Traits.Feature, Traits.Features.WinForms)]
         public void Add_Click_Handler()
         {
-            OpenFileWithDesigner(ProjectName, "Form1.vb");
+            OpenFileWithDesigner("Form1.vb", ProjectName);
             AddWinFormButton("SomeButton");
             EditWinFormsButtonEvent(buttonName: "SomeButton", eventName: "Click", eventHandlerName: "ExecuteWhenButtonClicked");
-            OpenFile(ProjectName, "Form1.vb");
+            OpenFile("Form1.vb", ProjectName);
             VerifyTextContains(@"Private Sub ExecuteWhenButtonClicked(sender As Object, e As EventArgs) Handles SomeButton.Click");
-            SaveFile(ProjectName, "Form1.vb");
+            SaveFile("Form1.vb", ProjectName);
```

Comment:
Should we make a little "Project" type of some kind to wrap this so we have better type safety?